includes/Admin/Admin_Page.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Admin\Admin_Page
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Admin;
+
+/**
+ * Class is handling admin tools page functionality.
+ *
+ * @since n.e.x.t
+ */
+class Admin_Page {
+
+	/**
+	 * Initialize hooks.
+	 */
+	public function add_hooks() {
+		add_action( 'admin_menu', array( $this, 'add_page' ) );
+	}
+
+	/**
+	 * Register the admin page under the tools menu.
+	 */
+	public function add_page() {
+		add_management_page( __( 'Plugin Check', 'plugin-check' ), __( 'Plugin Check', 'plugin-check' ), 'activate_plugins', 'plugin-check', array( $this, 'render_page' ) );
+	}
+
+	/**
+	 * Returns the list of plugins.
+	 *
+	 * @return array List of plugins.
+	 */
+	public function get_available_plugins() {
```

Comment:
This should not be a public method. It's not called anywhere else, and we should only make methods public if they have to be.